src/MSIdentityScaffolding/Microsoft.DotNet.MSIdentity/CodeReaderWriter/ProjectModifier.cs

```diff
@@ -0,0 +1,209 @@
+                            var classNode = namespaceNode?
+                                .DescendantNodes()?
+                                .Where(node =>
+                                    node is ClassDeclarationSyntax cds &&
+                                    cds.Identifier.ValueText.Contains(className))
+                                .FirstOrDefault();
+
+                            if (classNode != null && classNode is ClassDeclarationSyntax classDeclarationSyntax)
+                            {
+                                var modifiedClassDeclarationSyntax = classDeclarationSyntax;
+                                //adding usings
+                                documentBuilder.AddUsings();
+                                //add class properties
+                                modifiedClassDeclarationSyntax = documentBuilder.AddProperties(modifiedClassDeclarationSyntax);
+                                //add class attributes
+                                modifiedClassDeclarationSyntax = documentBuilder.AddClassAttributes(modifiedClassDeclarationSyntax);
+
+                                //add code snippets/changes.
+                                if (file.Methods != null && file.Methods.Any())
+                                {
+                                    modifiedClassDeclarationSyntax = documentBuilder.AddCodeSnippets(file, modifiedClassDeclarationSyntax); ;
+                                }
+                                //replace class node with all the updates.
+                                documentEditor.ReplaceNode(classDeclarationSyntax, modifiedClassDeclarationSyntax);
+                            }
+                            await documentBuilder.WriteToClassFile(fileName, filePath);
+                        }
+                    }
+                }
+            }
+        }
+
+        private CodeModifierConfig? GetCodeModifierConfig()
+        {
+            List<CodeModifierConfig> codeModifierConfigs = new List<CodeModifierConfig>();
+            if (!string.IsNullOrEmpty(_toolOptions.ProjectType))
+            {
+                var properties = typeof(Resources).GetProperties(BindingFlags.Static | BindingFlags.NonPublic)
+               .Where(p => p.PropertyType == typeof(byte[]))
+               .ToArray();
+
+                foreach (PropertyInfo propertyInfo in properties)
+                {
+                    if (propertyInfo.Name.StartsWith("cm") && propertyInfo.Name.Contains(_toolOptions.ProjectType))
+                    {
+                        byte[] content = (propertyInfo.GetValue(null) as byte[])!;
+                        CodeModifierConfig? projectDescription = ReadCodeModifierConfigFromFileContent(content);
+
+                        if (projectDescription == null)
+                        {
+                            throw new FormatException($"Resource file { propertyInfo.Name } could not be parsed. ");
```

Comment:
resources?

Reply:
@abpiskunov  fixed. There are a couple more instances of it but will fix in the next update.